src/codeMirror/index.jsx

```diff
@@ -0,0 +1,151 @@
+const CodeMirror = require('codemirror');
+const React = require('react');
+const PropTypes = require('prop-types');
+const Variable = require('@readme/variable');
+const { getMode } = require('../utils/modes');
```

Comment:
I assume loading these modes before `codemirror/addon/runmode/runmode` works just fine and languages like PHP and whatnot are still recognized?

Reply:
Indeed they do! Feel free to check it out when you have a sec!

Reply:
![Screen Shot 2020-09-14 at 2 02 23 PM](https://user-images.githubusercontent.com/33762/93137819-eec88c00-f692-11ea-81a5-bd1ecdb12d19.png)

sweet